docker/.env

```diff
@@ -37,17 +38,17 @@ LP_ETL_FUSEKI_ADMIN_PASSWORD=admin
 LP_ETL_FUSEKI_JVM_ARGS=-Xmx2g
 
 RDF_FINGERPRINTER_FUSEKI_DATA_FOLDER=fuseki-fingerprinter-volume
-RDF_FINGERPRINTER_FUSEKI_PORT=3020
-RDF_FINGERPRINTER_FUSEKI_LOCATION=http://fuseki-fingerprinter
+RDF_FINGERPRINTER_FUSEKI_PORT=3030
+RDF_FINGERPRINTER_FUSEKI_LOCATION=http://rdf-fingerprinter-fuseki
 RDF_FINGERPRINTER_FUSEKI_USERNAME=admin
 RDF_FINGERPRINTER_FUSEKI_PASSWORD=admin
 RDF_FINGERPRINTER_FUSEKI_ADMIN_PASSWORD=admin
 RDF_FINGERPRINTER_FUSEKI_JVM_ARGS=-Xmx2g
 RDF_FINGERPRINTER_API_PORT=4020
 RDF_FINGERPRINTER_UI_PORT=8020
 RDF_FINGERPRINTER_DEBUG=True
-RDF_FINGERPRINTER_API_LOCATION=http://fingerprinter-api
-RDF_FINGERPRINTER_UI_LOCATION=http://fingerprinter-ui
+RDF_FINGERPRINTER_API_LOCATION=http://rdf-fingerprinter-api
```

Comment:
I guess it is better with unified variable names, service names, container names (there are still some things to harmonise, but mostly acceptable)